advocacy_docs/partner_docs/ImpervaDataSecurityFabric/05-UsingImpervaDataSecurityFabric.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Using Imperva Agent with EDB Postgres Advanced Server'
+description: 'Walkthroughs of multiple Imperva usage scenarios'
+---
+
+This section features use cases that show how the Imperva Agent will monitor an EDB Postgres Advanced Server Database.  There are many different options and reports that can be configured with the Imperva Agent installed on your EDB Postgres Advanced Server or PostgreSQL database.  Some of these options are: scan and assess the servers, configure security policies, create and/or manage reports, configure and view audited traffic and configure or work with a Database Cluster.   For in depth information on the many pieces of Database Activity monitoring with the Imperva Agent follow the link: https://docs.imperva.com/bundle/v14.9-database-activity-monitoring-user-guide/page/70414.htm 
```

Comment:
Last sentence should end with "... please visit the Database Activity Monitoring User Guide on the Imperva website." and "Database Activity Monitoring User Guide on the Imperva website" should be hyperlinked to that link.